src/main.rs

```diff
@@ -371,7 +385,13 @@ pub fn handle_zsh_completion(matching_dirs: FrecentPathIter) {
 }
 
 fn handle_edit(cmd: &ArgMatches) -> PaziResult {
-    let mut frecency = load_frecency();
+    let mut frecency = match load_frecency() {
```

Comment:
The repetition of this match is icky. I can't use the `?` operator here because `PaziResult` isn't a real result type.

Our options are to impl `impl From<anyhow::Error> for PaziResult`, or to push error handling up one more layer and have this return a `Result`, and have the `PaziResult` handling code either use anyhow's downcasting, or otherwise dig into the error to behave correctly.

I'm planning to do this in a followup. We have several things (PaziResult, FilterError, PipeError) which I think all want similar behaviour, so I want to clean up those separately.